src/olympia/blocklist/utils.py

```diff
@@ -147,12 +147,15 @@ def disable_versions_for_block(block, submission):
     """Disable appropriate addon versions that are affected by the Block."""
     from olympia.reviewers.utils import ReviewBase
 
+    task_user = get_task_user()
```

Comment:
Reading the PR and issue it's not super clear to me why we have the fallback to `updated_by`. I also don't really understand why the review is considered human **only** if the `task_user` is not the user that last updated the block... this seems kind of arbitrary and really easy to miss or get wrong when looking at this code in 2 weeks let alone 2 years.. thoughts?

Reply:
`ActivityLog` needs a `user` no matter what, or it won't be saved (and that will be a silent error until https://github.com/mozilla/addons/issues/14803 is fixed).

If a block has no `updated_by` (which is technically possible, since the field is nullable/not required) we still want to record the activity. So it makes sense to fall back to the task user.

As a rule in addons-server, if the task user is responsible for something, then it was not a human action (see `auto_approve.py` or `auto_reject.py` for instance).